.gitignore

```diff
@@ -2,9 +2,7 @@ node_modules/
 build/
 log/
 
-config/config.development.json
-config/config.test.json
-config/config.production.json
+config/config.*.json
```

Comment:
@mingard Just added this because I have `config.development.local.json` and `config.development.remote.json`, which I rename based on whether I want to use my local API or the test one, and it'd be nice to also ignore these.